skeleton-navigation/src/resource.d.ts__if_typescript

```diff
@@ -0,0 +1,22 @@
+declare module '*.html' {
```

Comment:
Main as `src/main.ext`, can be deduplicated.

Reply:
Do you mean "Same as `src/main.ext` deduplicated"? 
If yes, done.